share/eds/accessor_cache.go

```diff
@@ -0,0 +1,106 @@
+package eds
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"sync"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+	lru "github.com/hashicorp/golang-lru"
+)
+
+var (
+	maxCacheSize = 128
```

Comment:
We need to have this configurable. Beefy nodes, like our bootstrappers, may want to have it much bigger

Reply:
Is now passed to the constructor, any more configurability (either through config or functional params) needs to be a later PR, after it is being supplied to FX (which doesn't happen here)

Reply:
👌